resources/build.js

```diff
@@ -0,0 +1,45 @@
+var execSync = require('child_process').execSync;
+var fs = require('fs');
+var rimraf = require('rimraf');
+var isWindows = /^win/.test(process.platform)
+
+var out = function (command) {
+  execSync(command, {stdio:'inherit'});
+};
+
+console.log('"Running node ./resources/build.js"');
+
+if (!isWindows) {
+  out('set -e'); // exit on console error
+} // put windows equivalent here if you know it
+
+if (!fs.existsSync('node_modules/.bin')) {
+  console.log('"Be sure to run \`npm install\` before building GraphiQL."');
+}
+
+var distDir = './dist';
+rimraf.sync(distDir);
+fs.mkdirSync(distDir);
+
+out('babel src --ignore __tests__ --out-dir dist/');
+console.log('"Bundling graphiql.js..."');
```

Comment:
They are accidental remnants from past echo commands. I can remove them, but noticed they seemed to help contrast between other console writes and made these easier to read even if the quotes lacked meaning. (There are JS filenames logged to console during the build without quotes.)